locust/dispatch.py

```diff
@@ -350,28 +347,31 @@ def _current_user_class_will_keep_distribution_better_than_all_other_user_classe
             # then the distribution will be the best we can get. In other words, adding
             # one user of any other user class won't yield a better distribution.
             return True
-        if distances.actual_distance_with_current_user_class <= distances.actual_distance:
+        if actual_distance_with_current_user_class <= actual_distance:
```

Comment:
Suggest you replace this with
return actual_distance_with_current_user_class <= actual_distance